CMakeLists.txt

```diff
@@ -329,7 +329,6 @@ message(STATUS "Example: ${SSG_PRODUCT_EXAMPLE}")
 message(STATUS "EKS: ${SSG_PRODUCT_EKS}")
 message(STATUS "Fedora: ${SSG_PRODUCT_FEDORA}")
 message(STATUS "Firefox: ${SSG_PRODUCT_FIREFOX}")
-message(STATUS "MacOS 1015: ${SSG_PRODUCT_MACOS1015}")
```

Comment:
Please sort this list as well. There is AL 2023 and OpenEmbedded at the end of the list.

Reply:
Done.